docs/glossary.md

```diff
@@ -0,0 +1,29 @@
+# Glossary
+
+## Account Servicing Entity
+
+An Account Servicing Entity provides and maintains a payment account for a payer and is a regulated entity in the country/countries it operates.
```

Comment:
payer and payee?

Reply:
Yes, I think we should add the payee. I took the definition from the [Open Banking](https://www.openbanking.org.uk/glossary/account-servicing-payment-service-provider/) people and adopted it to our case.